WG_JS_CORE.md

```diff
@@ -124,7 +126,12 @@ This milestone focuses in on how JS IPFS can refine and augment it’s existing
 
 ## Timeline
 
-- Q1
-- Q2
-- Q3
-- Q4
+`TODO`
```

Comment:
@alanshaw I believe I remember you telling me that this was already done. If so, can you add it here as well?